texascale-org/static/texascale-org/css/src/_imports/components/deprecated/c-offset-content.css

```diff
@@ -14,8 +14,6 @@ Styleguide Components.Texascale.Deprecated.OffsetContent
 [class*="c-offset-content--"] {
   --offset-distance: 8.5vw; /* NOTE: Value is from Texascale.org 2020 */
 	--buffer: 30px; /* double Bootstrap `.col` padding */
-
-  max-width: 50%;
```

Comment:
**Explanation.** This was causing offset content to be short width instead of full width.